lib/dnsimple/client/contacts.rb

```diff
@@ -23,8 +23,8 @@ def contacts(account_id, options = {})
 
         Dnsimple::PaginatedResponse.new(response, response["data"].map { |r| Struct::Contact.new(r) })
       end
-      alias :list :contacts
-      alias :list_contacts :contacts
+      alias list contacts
+      alias list_contacts contacts
```

Comment:
Yes, I knew, but I always disliked it. Unfortunately, it's also the recommended way (since `alias` it's a keyword and not a method), hence I decided to put aside my personal preference for the sake of following the standards. :smile: